### PR TITLE
refactor: `setBucketPolicy`

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

File: src/minio.js [hunk 1]
```diff
@@ -2283,3 +2254,5 @@ Client.prototype.getObjectTagging = callbackify(Client.prototype.getObjectTaggin
 Client.prototype.putObjectRetention = callbackify(Client.prototype.putObjectRetention)
 Client.prototype.setObjectLockConfig = callbackify(Client.prototype.setObjectLockConfig)
 Client.prototype.getObjectLockConfig = callbackify(Client.prototype.getObjectLockConfig)
+
+Client.prototype.setBucketPolicy = callbackify(Client.prototype.setBucketPolicy)
```
Comment (trim21) [inline src/minio.js:2258]:
```suggestion
Client.prototype.setBucketPolicy = callbackify(Client.prototype.setBucketPolicy)
```

**Review by aldy505 — COMMENTED** [review]
LGTM

**Comment by prakashsvmx** [comment]
Please resolve the conflicts. @trim21

File: examples/set-bucket-policy.js [hunk 2]
```diff
@@ -65,10 +65,5 @@ const policy = `
...
 
-s3Client.setBucketPolicy('testbucket', policy, (err) => {
-  if (err) {
-    throw err
-  }
-
-  console.log('Set bucket policy')
-})
+await s3Client.setBucketPolicy('testbucket', policy)
+console.log('Set bucket policy')
```
Comment (prakashsvmx) [inline examples/set-bucket-policy.js:69]: could you please rename the file to have `.mjs` extension.  and update `README.md`

**Review by prakashsvmx — APPROVED** [review]

**Comment by trim21** [comment]
@kaankabalak

**Pull request merged**